selfdrive/athena/athenad.py

```diff
@@ -317,6 +358,12 @@ def main():
   api = Api(dongle_id)
 
   conn_retries = 0
+
+  # Set up zmq socket
+  ctx = zmq.Context().instance()
+  sock = ctx.socket(zmq.PULL)
+  sock.bind("ipc:///tmp/logmessage")
```

Comment:
can zmq get disconnected (and we need to reconnect)?

Reply:
True, it might be possible. For the same reason, we should also fix this problem in `selfdrive/swaglog,py#22` (as it is now if the socket disconnects then all the log messages will be dropped silently)